packages/client/src/lib.js

```diff
@@ -108,61 +140,52 @@ class Web3Storage {
         'X-Name': name
       }
     }
+    console.log(1)
 
-    /** @type {string} */
-    let carRoot
-    const blockstore = new Blockstore()
-
-    try {
-      const { out, root } = await pack({
-        input: Array.from(files).map((f) => ({
-          path: f.name,
-          content: f.stream()
-        })),
-        blockstore,
-        wrapWithDirectory,
-        maxChunkSize: 1048576,
-        maxChildrenPerNode: 1024
-      })
-      carRoot = root.toString()
+    const [ root ] = await car.getRoots()
+    const carRoot = root.toString()
```

Comment:
```suggestion
    const [ root ] = await car.getRoots()
    if (root == null) {
      throw new Error('missing root CID')
    }
    const carRoot = root.toString()
```

Reply:
@alanshaw Accepted your suggestion and now it has noticed that there's no test coverage for those lines. Can you think of a way to make a CID so poorly formed that it'll fail in a test? Not sure if your build process will allow those lines not to be covered in tests (and what I'm seeing is just a suggestion) or if it will put up a stink.